WRITERS-NOTES.md

```diff
@@ -0,0 +1,232 @@
+| Tip                         | Green         | Green            |
+| Caution, Warning, Important | Amber         | Amber            |
+| Danger                      | Red           | Red              |
+| Summary                     | Turquoise     | Turquoise        |
+
+### Note, Info
+
+Use as a side panel, an 'aside', a note detached from the main flow of the text.
+
+Preferred use is without the label (first form).
+
+```txt
+!!! note alert alert-primary ""
+    Text ...
+
+!!! note alert alert-primary "Note"
+    Text ...
+
+!!! note alert alert-primary "Side topic"
+    Text ...
+```
+
+### Caution, Warning, Important
+
+Uses same type but different label text:
+
+- Caution: Used to mean 'Continue with care'. Less strong than 'Warning' IMHO.
+
+- Important: A significant point that deserves emphasis. (MkDocs default for 'important' admonition is green, which is why I don't use it.)
+
+Style:
+
+- MkDocs: Amber with triangle/! icon
+- Bootstrap: Yellow with no icon
+
+```txt
+!!! caution alert alert-warning "Caution"
+!!! caution alert alert-warning "Important"
+```
+
+### Danger
+
+Anything that has the potential to damage or compromise a user's data or system.
+
+- MkDocs: Red with bolt icon.
+- Boostrap: Red with no icon.
+
+```txt
+!!! danger alert alert-danger "Danger"
+```
+
+### Tip
+
+Use for tips, hints, non-essential but useful advice. Note that `tip` renders badly in Percona.com. `hint` is better and looks the same as `tip` in Material theme.
+
+```txt
+!!! hint alert alert-success "Tip"
+    Tip
+
+!!! hint alert alert-success "Tips"
+    - One
+    - Two
+
+!!! hint alert alert-success ""
+    Tip
+```
+
+### Summary
+
+Used to summarise a block of text (a TLDR).
+
+```txt
+!!! summary alert alert-info "Summary"
+```
+
+### See Also
+
+Used to highlight other sections or external links.
+
+Group them at the end of the section.
+
+An exception would be when there is an equivalent or closely related section elsewhere.
+
+- MkDocs: Blue with pen icon (same as note).
+- Bootstrap: Turquoise with no icon.
+
+```txt
+!!! seealso alert alert-info "See also"
+```
+
+## Variables
+
+We use the `mkdocs-macros` plugin for variable expansion. For example, the variable `release` in `variables.yml` is used in the code so that the current PMM release number is always up-to-date. (Search the markdown files for `{{release}}`.)
+
+This plugin can have problems when Jinja-like constructs are used in code. This happens when referring to Docker variables. Workarounds are explained here: https://github.com/fralau/mkdocs_macros_plugin/blob/master/webdoc/docs/advanced.md#solutions
+
+In some places, we have used variables themselves to solve the problem. In others, `{% raw %}/{% endraw %}` surrounds the conflicting text.
+
+## Icons
+
+Use HTML for icons:
+
+- For PMM UI (Grafana) icons (`uil-` prefix), go to <https://iconscout.com/unicons/explore/line>, find an icon, select 'Font' and copy the code here.
```

Comment:
we need to check if icons on iconscout are relevant for Everest

Reply:
@catalinaadam The best is to ask the FE developer. If negative, we can remove these for the sake of speed.